### ISSUE TITLE
Make Python interface more Pythonic

### ISSUE THREAD

**Issue by kdheepak** [GMLC-TDC/HELICS#44]

**Comment by phlptp** [comment]
The full python test suite should be in place before we get much further in the development of this

**Comment by phlptp** [comment]
I think some of the changes using swig have made this a little less of a priority so I am moving it to HELICS 2.0 tag, as I think once we revisit the C-api  we will want to to layer an object Oriented pthyon code layer on top of it to really make the interface easy to use.

**Comment by phlptp** [comment]
@kdheepak  do you think this is practical by Mid July.  It would be nice to have in place before actually shifting to the 2.0 branch so the pythonic API doesn't change when we make the major changes in the underlying C API

**Comment by phlptp** [comment]
@kdheepak what is the status of this?  Is it ready or is it going to be part of the 2.0 series?

**Comment by trevorhardy** [comment]
@kdheepak: I think you get to check this one as complete, right?

**Comment by phlptp** [comment]
@kdheepak  I think you can close this one when you get a chance

**Comment by kdheepak** [comment]
See https://python.helics.org for more information.
